From d05ba1e2208cb17b8cf7dac050d95137a67dd988 Mon Sep 17 00:00:00 2001
From: Eric Auger <eric.auger@redhat.com>
Date: Thu, 24 Jun 2021 10:32:08 +0200
Subject: [PATCH 01/12] aarch64-rh-devices: add CONFIG_PXB
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Eric Auger <eric.auger@redhat.com>
RH-MergeRequest: 14: aarch64-rh-devices: add CONFIG_PXB
RH-Commit: [1/1] 6a9e6a96ea6ba1bee220a60e5a691a174a0a044b (eauger1/centos-qemu-kvm)
RH-Bugzilla: 1967502
RH-Acked-by: Gavin Shan <gshan@redhat.com>
RH-Acked-by: Daniel P. Berrangé <berrange@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>

We want to enable the PCIe expander bridge on aarch64. So let's
set CONFIG_PXB.

Signed-off-by: Eric Auger <eric.auger@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/devices/aarch64-rh-devices.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/default-configs/devices/aarch64-rh-devices.mak b/default-configs/devices/aarch64-rh-devices.mak
index 4220469178..d8ce902720 100644
--- a/default-configs/devices/aarch64-rh-devices.mak
+++ b/default-configs/devices/aarch64-rh-devices.mak
@@ -27,3 +27,4 @@ CONFIG_TPM_TIS_SYSBUS=y
 CONFIG_PTIMER=y
 CONFIG_ARM_COMPATIBLE_SEMIHOSTING=y
 CONFIG_PVPANIC_PCI=y
+CONFIG_PXB=y
-- 
2.27.0

